### ISSUE TITLE
resource use creation

### ISSUE THREAD

**Issue by GiselleSerate** [studycloud/studycloud#119]
Like #117, is a search of all valid resource uses in a drop-down. Choose one use exactly.

**Comment by aryam7** [comment]
I'm imagining the following JSON format for this. Let me know if you have any suggestions/improvements.
```
{
    data: [
        {
            id: 1,
            name: "Class Notes"
        },
        {
            id: 2,
            name: "Reading Notes"
        },
        {
            id: 3,
            name: "Flashcards"
        }
    ]
}
```

**Comment by aryam7** [comment]
Wait, we decided you would need this on page load only, right? So we aren't going to create an AJAX request for it?

In that case, you can actually grab this data super easily. Just stick this HTML code into every blade file in which you need the data.
```
<script>
resource_uses = @json( App\ResourceUse::select('id', 'name')->get() );
</script>
```
Note that it needs to be included before the JS that uses it.

It's equivalent to doing this in your JS:
```
resource_uses = [
    {
        id: 1,
        name: "Class Notes"
    },
    {
        id: 2,
        name: "Reading Notes"
    },
    {
        id: 3,
        name: "Flashcards"
    }
];
```

**Comment by GiselleSerate** [comment]
Also allow people to create their own uses.

**Comment by aryam7** [comment]
To get all possible content types:
```
<script>
resource_types = @json( App\ResourceContent::getPossibleTypes() );
</script>
```

**Comment by aryam7** [comment]
resolved in #172
